frontend/src/components/pages/GlobalForms/WaiversTab.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiverSectionCard from "./WaiverSectionCard";
+  
+  interface WaiversTabProps {
+    clauses: Array<ClauseInfo>;
+  }
+  const WaiversTab = ({
+    clauses
+  }: WaiversTabProps): JSX.Element => {
+
+    function intToChar(number: number) {
+        const code = 'A'.charCodeAt(0);  
+        return String.fromCharCode(code + number);
+    }
+    return (
+        <>
+        {clauses.length && (clauses).map((clause, idx)=> {
+            return <WaiverSectionCard key={idx} title={`Section ${intToChar(idx)}`}
+            waiverText={clause.text}
```

Comment:
Instead of passing in each property of the clause, can you just pass in the whole clause? This would just make code a little cleaner

Reply:
so you only need to pass in the clause and the index to the `WaiverSectionCard` component. The rest can be handled inside the component itself